src/main/java/seedu/address/logic/commands/AbstractEditCommand.java

```diff
@@ -66,6 +66,7 @@ public CommandResult execute(Model model) throws CommandException {
         init();
         editFields();
         validateEditedAndWriteBack();
+        model.resetAllShowFields();
```

Comment:
Thank you for working on this; I also touch this on #140, but for add or edit command will set the show to be the newly added entry, for delete command will set show to null if the current shown entry is deleted, and update the ui.